src/main/resources/db/migration/common/V1_218__node_register_missing_fk_and_seq.sql

```diff
@@ -0,0 +1,18 @@
+-- v47
+-- changes were not kept update to date for the node_register table so its being done now with the new
+-- changes for the table in v47.
+CREATE SEQUENCE node_seq START WITH 10000;
+
+ALTER TABLE node_register
```

Comment:
I think this table should be called `device_register`?

Reply:
Talking to Barry he does not have a preference for this table and it seems like Sashi made a request for it to be node register. So I have decided to keep it as it was previously. Barry needs to change his script.

Reply:
Fingers crossed

Reply:
Going to keep these changes as has been discussed with Sashi and Barry. The new proposed changes in v50 CREATE TABLE node_register
(node_id                     INTEGER                       NOT NULL  --reusing PK from legacy
,ctr_id                      INTEGER                       NOT NULL
,node_type                   CHARACTER VARYING             NOT NULL   DEFAULT 'DAR'
,hostname                    CHARACTER VARYING             NOT NULL
,ip_address                  CHARACTER VARYING             NOT NULL
,mac_address                 CHARACTER VARYING             NOT NULL
,created_ts                  TIMESTAMP WITH TIME ZONE      NOT NULL
,created_by                  INTEGER                       NOT NULL
,last_modified_ts            TIMESTAMP WITH TIME ZONE      NOT NULL
,last_modified_by            INTEGER                       NOT NULL
) TABLESPACE darts_tables;